packages/engine-test-utils/src/__tests__/engine-server/enginev2.spec.ts

```diff
@@ -173,3 +173,20 @@ describe("engine, config/", () => {
     });
   });
 });
+
+test("WHEN querying for a note that doesn't exist, return empty array in data", async () => {
```

Comment:
can you use the engine test harness as described here? [[Testing a new Engine Method|dendron://dendron.docs/pkg.engine-test-utils.qa.test#testing-a-new-engine-method]]

the advantage is that it will test the engine both directly and via api call

Reply:
gotcha, looks good